app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+    @comment = @post.comments.new
+  end
+
+  def create
+    @comment = current_user.comments.new(comment_params)
+
+    # puts @comment
+    if @comment.save
+      redirect_to user_post_path(current_user, params[:post_id])
+    else
+      puts @comment.errors.messages
+      render :new
+    end
+  end
+
+  private
+
```

Comment:
- [ ] I can see you don't have like controller at all, whereas, a user should be able to add a like to posts.
The likes controller should have a create actions similar to the ones you've created. By doing that, it would create the route too.
![image](https://github.com/GeekyHacks/FlightOperationsBlog/assets/86200356/afeff58e-5bfb-4248-ae16-49ee65f94458)